extension/apmconfigextension/extension.go

```diff
@@ -36,6 +37,14 @@ const (
 	defaultOpAMPPath = "/v1/opamp"
 )
 
+var defaultCacheConfig = cacheConfig{
+	// Cache capacity for active agents
+	capacity: 1000,
```

Comment:
nit: use 1024 instead of 1000 because computers work best with 2^N :)

Reply:
Good point, let's be friendly with computers then :) https://github.com/elastic/opentelemetry-collector-components/pull/629/commits/308f6995550b189eac6e2a6f958f0dd6de7914d0